Webpack 5 and node core modules (fs, path)

If you try to bundle this file 

```js
import { getHighlighter } from "shiki"
```

with Webpack 5 targeting a browser (webpack's default). You get this error:

```error
...

ERROR in ./node_modules/shiki/dist/index.esm.js 2129:25-40
Module not found: Error: Can't resolve 'path' in 'node_modules/shiki/dist'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

...

ERROR in ./node_modules/shiki/dist/index.esm.js 2131:23-36
Module not found: Error: Can't resolve 'fs' in 'node_modules/shiki/dist'

...
```

One solution is to add the `browser` field to shiki's package.json (I'll send a PR), see https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-nodejs-polyfills-removed.

---

Repro:

`package.json`
```json
{
  "devDependencies": {
    "webpack": "^5.45.1",
    "webpack-cli": "^4.7.2"
  },
  "dependencies": {
    "shiki": "^0.9.5"
  }
  ,
  "scripts": {
    "build": "webpack"
  }
}
```

`src/index.js`
```js
import { getHighlighter } from "shiki";
```

Then run `yarn && yarn build`

--- a/packages/shiki/package.json
+++ b/packages/shiki/package.json
@@ -34,5 +34,9 @@
     "json5": "^2.2.0",
     "onigasm": "^2.2.5",
     "vscode-textmate": "5.2.0"
+  },
+  "browser": {
+    "fs": false,
+    "path": false
   }
 }

